sys/net/gnrc/netif/gnrc_netif_device_type.c

```diff
@@ -287,6 +287,11 @@ int gnrc_netif_ndp_addr_len_from_l2ao(gnrc_netif_t *netif,
                     return -EINVAL;
             }
 #endif  /* defined(MODULE_NETDEV_IEEE802154) || defined(MODULE_XBEE) */
+#if defined(MODULE_CC110X)
+        case NETDEV_TYPE_CC110X:
+            (void)opt;
+            return sizeof(uint8_t);
+#endif  /* defined(MODULE_CC110X) */
```

Comment:
Above added code seems to be added by accident, e.g. error during a merge. (See lines 257-261 above, the contain the same code. This should not compile with `cc110x` because of the duplicated case.)